.github/workflows/build-container.yml

```
@@ -0,0 +1,85 @@
+name: Build and Push Docker image
+
+on:
+  push:
+    branches: [ master ]
+  repository_dispatch:
+    types:
+      - dispatch-build
+  workflow_dispatch:
+
+jobs:
+  make-date-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+    steps:
+    - name: make date tag
+      id: mkdatetag
+      run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
+
+  build:
+    runs-on: ubuntu-latest
+    needs: [make-date-tag]
+    strategy:
+      fail-fast: False
+      matrix:
+        dver: ['7', '8']
+        repo: ['devel', 'testing', 'release']
+        series: ['3.5', '3.6']
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Generate tag list
+      id: generate-tag-list
+      env:
+        EL_VER: ${{ matrix.dver }}
+        BASE_YUM_REPO: ${{ matrix.repo }}
+        OSG_RELEASE: ${{ matrix.series }}
+        TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
+      run: |
+        docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
+        tags=()
+        for registry in hub.opensciencegrid.org docker.io; do
+          tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO-el$EL_VER )
+          if [[ $EL_VER == '8' ]]; then
+            tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO )
+            if [[ $BASE_YUM_REPO == 'release' ]]; then
+              tags+=( $registry/$docker_repo:$OSG_RELEASE )
+              [[ $OSG_RELEASE == '3.6' ]] && tags+=( $registry/$docker_repo:latest )
+            else
+              [[ $OSG_RELEASE == '3.6' ]] && tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
+            fi
+          elif [[$BASE_YUM_REPO == 'release' ]]; then
```

Oops, using an `else` or `elif` here meant that we'd actually skip the `3.6-el8` and `3.5-el8` tags. That's why you had it above! This should work and avoid the duplication:

```suggestion
          if [[$BASE_YUM_REPO == 'release' ]]; then
```